sections/sprint-1/networking.tex

```diff
@@ -269,10 +269,18 @@ \subsection{IPv4 and IPv6}\label{ipv4-ipv6}
 \subsection{Format of data}\label{subsec:data-format}
 As it is necessary to send data over the network, the format of the data must be decided upon.
 The network format for the field anchor, player position and goal position is \texttt{0xyyyyxxxxiisstt}, where \texttt{t} is the type, \texttt{s} is the timestamp, \texttt{i} is the ID, \texttt{xxxx} is the \texttt{x} position and \texttt{yyyy} is the \texttt{y} position.
-These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:player-format}.
+These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:goal-position-format}.
 The data sent is almost identical, as the same information is required, the only difference between them is the type and the context of the IDs.
+The \texttt{x} and \texttt{y} position is the location of the player or anchor.
+For \autoref{Tab:goal-position-format} the \texttt{x} and \texttt{y} position is the center of the goal, where the size of the goal is set in unity.
+The IDs for \autoref{Tab:anchor-format} is for each anchor.
+For \autoref{Tab:player-format} the ID 0 is the ball, where 1-4 is a player and for \autoref{Tab:goal-position-format} the ID identifies what team it is..
 The type is at the right hand side of the hexadecimal so that it is faster to decode and to switch on which type it is.
 After the type the timestamp is placed, which is used to detect outdated data.
+The timestamp is an integer that represents time from 0-255 and when it hits 255 it loops to 0 again as the newest value.
+When implementing this, it is required to consider if some packages are not received.
+A solution to this could be that 0 to 20 is considered as newer timestamps than 235 to 255 as it is very unlikely that 20 packages are not receive 20 times in a row.
```

Comment:
```suggestion
The downside to this approach is since UDP does not guarantee delivery of messages, it may not go straight from 255 to 0 every time.
To counter this, 0 to 20 is considered as newer timestamps than 235 to 255 so that the system is allowed to miss up to 20 messages near the end, to ensure that it does not end up being locked until the values reach the high range again.
```